### ISSUE TITLE
please pass request param to django.contrib.auth.authenticate

### ISSUE THREAD

**Issue by mqinbin** [GetBlimp/django-rest-framework-jwt#455]
for some reason i want to use "request" in AUTHENTICATION_BACKENDS
but i got request as None

i need request because:
1.  shopping cart is in cookie before login
2. shoping cart is in server after login
3. i want to decorate anywhere when login ( logon 、login 、oauth2)
4. in my decoration : i get user from request 
5. i write a new AUTHENTICATION_BACKENDS give User to request as user property
no request no user

just change you serializers line 50
 from
           user = authenticate(**credentials)
to
           user = authenticate(self.context["request"] , **credentials)

thanks

**Comment by ehmadzubair** [comment]
@jpadilla If i create a PR, will this be merged?

**Comment by string-areeb** [comment]
No, most likely not. Project hasn't been updated for 2 years. Use https://github.com/davesque/django-rest-framework-simplejwt/ instead
